docs/cookbook/events_and_gestures.md

```diff
@@ -107,6 +107,19 @@ end
 
 ```
 
+## Custom events
+
+To add a custom event, use `.on` with a custom symbol. Call `.trigger` to trigger your block.
+
+```ruby
+
+# this code allows you to add a custom event and trigger it
+rmq.append(UIView, :my_view).on(:custom_event) do |sender|
+    puts "custom_event has been triggered"
+end
+rmq(:my_view).trigger(:custom_event)
```

Comment:
probably a question for @twerth, I just assume that in redpotion docs we'd take advantage of the redpotion helper stuff - granted this works.  You definitely dont have to close this PR you could just ammend your commit if you wanted to update it :+1: